GRIB2_Template_4_144_ProductDefinitionTemplate_en.csv

```diff
@@ -0,0 +1,39 @@
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,51,1,Minute - time of end of overall time interval,,,,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,52,1,Second - time of end of overall time interval,,,,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,53,1,n - number of time range specifications describing the time intervals used to calculate the statistically processed field,,,,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,54-57,4,Total number of data values missing in statistical process,,,,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,,,58-69   Specification of the outermost (or only) time range over which statistical processing is done,,,,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,58,1,Statistical process used to calculate the processed field from the field at each time increment during the time range,(see Code table 4.10),,4.10,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,59,1,Type of time increment between successive fields used in the statistical processing,(see Code table 4.11),,4.11,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,60,1,Indicator of unit of time for time range over which statistical processing is done,(see Code table 4.4),,4.4,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,61-64,4,"Length of the time range over which statistical processing is done, in units defined by the previous octet",,,,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,65,1,Indicator of unit of time for the increment between the successive fields used,(see Code table 4.4),,4.4,,Operational
+Analysis or forecast at a horizontal level or in a horizontal layer in a continuous or non-continuous time interval for waves selected by period range,66-69,4,"Time increment between successive fields, in units defined by the previous octet",(see Notes 3 and 4),"7,96",,,Operational
```

Comment:
are these notes correct? 
3) Above normal, near normal and below normal are defined as three equiprobable categories based on climatology at each point over the geographical area covered by the grid. The type and methodology of the reference climatology are unspecified and should be documented concurrently by the data producer.
(4) The original data value is defined by Code table 4.244.

Reply:
These are the notes from the right template notes table for (7,96)

(3) An increment of zero means that the statistical processing is the result of a continuous (or near-continuous) process, not the processing of a number of discrete samples. Examples of such continuous processes are the temperatures measured by analogue maximum and minimum thermometers or thermographs, and the rainfall measured by a rain gauge.
(4) The reference and forecast times are successively set to their initial values plus or minus the increment, as defined by the type of time increment (one of octets 48, 60, 72, ...). For all but the innermost (last) time range, the next inner range is then processed using these reference and forecast times as the initial reference and forecast times.